### PR TITLE
#165305712 Integrating Travis-Ci with GitHub

### PR DESCRIPTION
**Pull request by a-braham**
### What does this PR do?
Integrates Travis-ci to GitHub
### What are the tasks to be completed?
- Add Travis Ci badge

### How can this be manually tested?
After cloning the repository:
```
$ git checkout ch-travis-ci-165305712
$ npm run start:dev
```

#### What are the relevant pivotal tracker stories?
[#165305712](https://www.pivotaltracker.com/n/projects/2326460/stories/165305712)

### CODE REVIEW

**Review by IssaIan — APPROVED** [review]
LGTM

**Review by Ogutu-Brian — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm install
+
+script:
+  - npm run coverage
+  - npm run test
```
Comment (Bakley) [inline .travis.yml]: Add a new line here

**Review by Bakley — APPROVED** [review]

**Review by Philipotieno — APPROVED** [review]
Everything looks good you could consider squashing the commits before the PR is closed.

File: package.json [hunk 2]
```diff
@@ -4,10 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```
Comment (koechkevin) [inline package.json]: @a-braham to avoid redundancy lets have this command as `"jest --coverage"` and do away with line 11

Comment (koechkevin) [reply package.json]: we would then have scripts in `.travis.yml` as only `npm run test`

**Review by koechkevin — CHANGES_REQUESTED** [review]

**Review by koechkevin — APPROVED** [review]

**Comment by koechkevin** [comment]
LGTM

**Pull request merged**